app/services/supplementary-billing/fetch-previous-billing-transactions.service.js

```diff
@@ -72,7 +72,12 @@ async function _fetch (licenceId, invoiceAccountId, financialYearEnding) {
       'bt.volume',
       'bt.section126Factor',
       'bt.section127Agreement',
-      'bt.section130Agreement',
+      // NOTE: This field is a varchar in the DB for historic reasons. It seems some early PRESROC transactions
+      // recorded values other than 'true' or 'false'. For SROC though, it will only ever be true/false. We generate
+      // our calculated billing transaction lines based on the Section130 flag against charge_elements which is always
+      // a boolean. So, to avoid issues when we need to compare the values we cast this to a boolean when fetching the
+      // data.
```

Comment:
Is it worth adding the literal text `section130Agreement` to the comment to make it findable when searching the codebase?
```suggestion
      // NOTE: The section130Agreement field is a varchar in the DB for historic reasons. It seems some early PRESROC
      // transactions recorded values other than 'true' or 'false'. For SROC though, it will only ever be true/false. We
      // generate our calculated billing transaction lines based on the Section130 flag against charge_elements which is
      // always a boolean. So, to avoid issues when we need to compare the values we cast this to a boolean when
      // fetching the data.
```